src/Loaders/Loaders.story.tsx

```diff
@@ -0,0 +1,112 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import { storiesOf } from "@storybook/react";
+import { LoadingSpinner, Size } from "./LoadingSpinner";
+import { Button } from "../Button";
+import { colors } from "../colors";
+import * as typography from "../typography";
+import { DemoSection, DemoGroup } from "../shared/DemoSection";
+
+const SPINNER_SIZES: Size[] = ["large", "medium", "small", "tiny", "icon"];
+
+storiesOf("Loaders", module)
+  .addParameters({
+    options: {
+      showPanel: false,
+    },
+  })
+  .add("Catalog", () => (
+    <div css={{ color: colors.black.base }}>
+      <DemoSection
+        title="Spinners"
+        description="Spinners are used when we are unable to determine loading time. Ideally, they should appear as briefly and infrequently as possible."
+      >
+        <DemoGroup
+          title="On Pages & Cards"
+          description="Spinners will most often appear on full pages and cards. Whenever possible, they should be paired with descriptive text that helps the user understand exactly what the system is working toward. Spinners should be center-aligned to the page with any text spaced 20 px below."
+        >
+          <div
+            css={{
+              flex: "1 1 0%",
+              border: `1px solid ${colors.silver.dark}`,
+              borderRadius: 8,
+              margin: 6,
+            }}
+          >
+            {SPINNER_SIZES.map(size => (
+              <div key={size} css={{ margin: 20 }}>
+                <span
+                  css={{
+                    ...typography.base.small,
+                    fontWeight: 600,
+                    display: "block",
+                    textTransform: "uppercase",
+                  }}
+                >
+                  {size}
+                </span>
+                <LoadingSpinner css={{ display: "block" }} size={size} />
```

Comment:
Does this `css` actually do anything considering that you don't accept a `className` prop in `LoadingSpinner`? 

I started moving towards using the more verbose and less magical `<ClassNames>{({css, cx}) => (<div className={cx(css({ display: 'block' }))} />))}</ClassNames>` because then `className`s are passed through as expected. `css` was causing me a lot of issues when trying to use `cloneElement()` (or `jsx()`, which is _not_ the same thing as `cloneElement()` because it skips `key` and `ref`s)

Reply:
`LoadingSpinner` still accepted a `className` (passing through `otherProps`). I can rewrite this using `ClassNames` if thats the way we want to go but personally I find it less readable/ergonomic compared to `css`.

Reply:
I wasn't suggesting `ClassNames`, just mentioning that I had to use it elsewhere. I also find it less ergonomic, but sometimes it's the only way to get things done.

I missed that you were passing through the other props. 👍